### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Random-Liu**
Update README.md.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,15 @@
...
 `cri-containerd` is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
+
+With it, you could run Kubernetes using containerd as the container runtime.
 ![cri-containerd](./docs/cri-containerd.png)
 ## Current Status
-`cri-containerd` is in alpha. This release is for use with Kubernetes 1.8. See
-the [roadmap](./docs/proposal.md#roadmap-and-milestones)
-for information about current and future milestones.
+`cri-containerd` is in alpha:
+* It is feature complete;
```
Comment (mikebrow) [inline README.md]: periods instead of semicolons..

File: README.md [hunk 2]
```diff
@@ -8,11 +8,15 @@
...
+* It works with Kubernetes >= 1.7;
+* It has passed all [CRI validation test](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-validation.md);
```
Comment (mikebrow) [inline README.md]: /test/tests/

File: README.md [hunk 3]
```diff
@@ -8,11 +8,15 @@
...
+* It has passed all regular [node e2e test](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-node-tests.md).
```
Comment (mikebrow) [inline README.md]: /test/tests/

File: README.md (hunk 1 above)
Comment (Random-Liu) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Random-Liu) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Random-Liu) [reply README.md]: Done

**Comment by Random-Liu** [comment]
@mikebrow Done.

**Pull request merged**